spec/controllers/topics_spec.rb

```diff
@@ -40,7 +40,7 @@
       let(:data) {{
         type: 'create',
         url:  'https://example.com/widgets/123',
-        timestamp: Time.now.to_f
+        timestamp: Time.now.to_i * 1e3
```

Comment:
Looks like you're switching from seconds to milliseconds, but you're also dropping precision. Is it required by Oj?